src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh

```diff
@@ -2,11 +2,13 @@
 
 set -ex
 
+kcli='docker run --net host -it --rm --security-opt label=disable -v /home/athakkar/.ssh:/root/.ssh -v /home/athakkar/.kcli:/root/.kcli -v /var/lib/libvirt/images:/var/lib/libvirt/images -v /var/run/libvirt:/var/run/libvirt -v /home/athakkar/ceph:/workdir quay.io/karmab/kcli'
+
 cleanup() {
     set +x
     if [[ -n "$JENKINS_HOME" ]]; then
         printf "\n\nStarting cleanup...\n\n"
-        kcli delete plan -y ceph || true
+        $kcli delete plan -y ceph || true
```

Comment:
these too.

Reply:
I have to squash some commits anyways..so will get rid of these changes as well.